### PR TITLE
Feature/message status alert

### PR DESCRIPTION
**Pull request by bhohnco**
## Type of Change
- [ ] JSX
- [ ] HTML
- [X] JavaScript
- [X] CSS
- [ ] React
- [ ] React Router
- [ ] Cypress
- [ ] Apollo
- [ ] GraphQL
- [X] Twilio API
- [ ] Refactor/Fix Bugs

## Please describe the change made
Happy and sad path error handling for text message status 

## Please give a short description of the changes that were made below

- Fix:

- Feature:
The user now gets pop up notifications when a text message is sent within the user flow
All 4 messages have happy and sad path notifications and have been tested 
The pop-up message is generated based on the Twilio API call and if it completes or fails. 

## How has this been tested?
- [X] In Browser
- [ ] In Console
- [ ] Cypress
- [ ] Postman
- [X] Mobile

Screen Shot 2021-07-20 at 1 34 51 PM

Screen Shot 2021-07-20 at 1 36 34 PM

Screen Shot 2021-07-20 at 1 37 07 PM

Screen Shot 2021-07-20 at 1 37 26 PM

### CODE REVIEW

File: src/Components/App/App.js [hunk 1]
```diff
@@ -23,13 +25,15 @@ function App() {
           <LoginView />
         </Route>
         <Route exact path='/'>
+          <Popup />
```
Comment (pcmueller) [inline src/Components/App/App.js:28]: Excited to see this new component in action!

File: src/Components/TripExtendedMessage/TripExtendedMessage.js [hunk 2]
```diff
@@ -16,8 +17,16 @@ export function TripExtendedMessage(user, extension, contact) {
...
-        .then(result => result.text())
-        .then(resp => console.log(resp))
+        .then(response => {
+          if(response.status === 201) {
+            console.log(response.status)
+            Popup.alert(`Trip extended notification successfully sent to ${contact.value}!`)
+            return response.text();
+          } else {
+            console.log("API ERROR")
+            Popup.alert(`Trip extended notification to your contact was unsuccessful, please contact ${contact.value}.`)
```
Comment (pcmueller) [inline src/Components/TripExtendedMessage/TripExtendedMessage.js:27]: Solid work with the error handling.

**Review by pcmueller — APPROVED** [review]
Nice work with the popup modal and SMS error handling!

File: src/Components/TripCompleteMessage/TripCompleteMessage.js [hunk 3]
```diff
@@ -16,8 +16,14 @@ export function TripCompleteMessage(contact, user) {
       },
       body: JSON.stringify(smsObj)
     })
-        .then(result => result.text())
-        .then(resp => console.log(resp))
+        .then(response => {
+          if(response.status === 201) {
+            Popup.alert(`Trip complete notification successfully sent to ${contact.value}! Thank you for using Walk Safe!`)
```
Comment (carolineshowscode) [inline src/Components/TripCompleteMessage/TripCompleteMessage.js:21]: These look awesome, great job figuring this out!

**Review by carolineshowscode — APPROVED** [review]
Nice work!

**Review by gaelyn — APPROVED** [review]
gif

**Pull request merged**